<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewKubeConfigDialog</class>
 <widget class="QWidget" name="CreateNewKubeConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelFolder">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditFolder">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFilename">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditFilename"/>
   </item>
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
opacity: 0</string>
     </property>
     <property name="text">
      <string>**ERROR**</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
